.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'kind/bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+<!--
+Steps to reproduce the behavior
+
+If possible add a minimal reproducer code sample
+
+-->
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Log/Stacktrace**
+
+<!-- If possible add the full stacktrace or log which contains the issue. -->
+
+<details><summary>Full Stacktrace</summary>
+ <p>
+
+```
+<STACKTRACE>
+```
+
+</p>
+</details>
+
+**Environment:**
+- OS: <!-- [e.g. Linux] -->
+- Deployment: <!-- [SaaS or SM (Self Managed)] --> 
+- Zeebe Version: <!-- [e.g. 0.20.0] -->
+- SpringBoot Version: <!-- [e.g. 2.7.x, 3.x.x] --> 
```

Comment:
These 2 might be clumped under `Configuration` or can be left as standalone. Could also add JDK version as well but that might be a too much as this list will be growing